pkg/dockerregistry/server/registry_test.go

```diff
@@ -36,15 +36,14 @@ func newTestRegistry(
 	osClient registryclient.Interface,
 	storageDriver driver.StorageDriver,
 	blobrepositorycachettl time.Duration,
-	pullthrough bool,
 	useBlobDescriptorCacheProvider bool,
 ) (distribution.Namespace, error) {
 	cfg := &configuration.Configuration{
 		Server: &configuration.Server{
 			Addr: "localhost:5000",
 		},
 		Pullthrough: &configuration.Pullthrough{
-			Enabled: pullthrough,
+			Enabled: true,
```

Comment:
Can we just drop this parameter from the configuration?

Reply:
@dmage I want to show warning if someone have it as `false`. But I can drop the `Pullthrough` initialization here.